packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -533,19 +615,12 @@ describe("toMapTree", () => {
 		});
 
 		it("List containing `undefined` (throws if fallback type is not allowed by the schema)", () => {
-			const schemaBuilder = new SchemaBuilder({ scope: "test" });
-			const rootSchema = schemaBuilder.list("test-list", [schemaBuilder.number]);
-			const schema = schemaBuilder.intoSchema(rootSchema);
+			const schemaFactory = new SchemaFactory("test");
+			const schema = schemaFactory.array("test-list", [schemaFactory.number]);
```

Comment:
you can use a structually named schema, and implicit allowed types:
```suggestion
			const schema = schemaFactory.array(schemaFactory.number);
```

Inlineing that below would be useful:
assert.throws(() => nodeDataToMapTree(input as InsertableContent, [schemaFactory.array(schemaFactory.number)]));
nodeDataToMapTree(input as InsertableContent, [schemaFactory.array([schemaFactory.number, schemaFactory.null])]);